spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -32,34 +34,28 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
       links = document.createElement('div')
       links.innerHTML =
           '<div class="fully-structured-external-links">' +
-              '<a href="http://www.nationalarchives1.gov.uk"> National Archives </a>' +
-              '<a href="https://www.nationalarchives2.gov.uk"></a>' +
-              '<a href="https://www.nationalarchives3.gov.uk/one.pdf">National Archives PDF</a>' +
+              '<a href="http://www.nationalarchives.gov.uk/1" link_domain="http://www.nationalarchives.gov.uk"> National Archives </a>' +
```

Comment:
Might be worth adding a comment here to explain why the `link_domain` attribute is here and that it's not expected in normal usage.